features/node/nto.feature

```diff
@@ -0,0 +1,92 @@
+Feature: Node Tuning Operator related scenarios
+
+  # @author skordas@redhat.com
+  # @case_id OCP-27491
+  @admin
```

Comment:
@skordas I think this is a destructive test?

Reply:
No, is not :)

Even test will pass or fail, cluster should be 100% operational. Additional step `Given admin ensures "nf-conntrack-max" tuned is deleted from the "openshift-cluster-node-tuning-operator" project after scenario` should remove custom NTO resource after test.
NTO is a Second Line Operator - OpenShift will work correctly even NTO will be deactivated.
In worse case scenario - kernel in one of working node will be working with different `nf-conntrack-max` value (and is only 2 bytes from 1Mb)

Reply:
so, as long as we can run them in parallel.